test_scripts/UpdateVideoStreamingCapabilities/common.lua

```diff
@@ -219,6 +267,43 @@ function m.startVideoService(pData, pAppId, isEncryption)
     end)
 end
 
+function m.startSecureVideoService(pData, pTimes)
+  local videoData = getVideoDataForStartServicePayload(pData)
+
+  local videoPayload = {
+    height          = { type = bsonType.INT32,  value = videoData.height },
+    width           = { type = bsonType.INT32,  value = videoData.width },
+    videoProtocol   = { type = bsonType.STRING, value = videoData.videoProtocol },
+    videoCodec      = { type = bsonType.STRING, value = videoData.videoCodec },
+  }
+
+  sslHandshakeFlow.getMobileSession():StartSecureService(constants.SERVICE_TYPE.VIDEO, bson.to_bytes(videoPayload))
```

Comment:
@GetmanetsIrina Please use `actions.mobile.getSession` instead of `sslHandshakeFlow.getMobileSession()`

Reply:
Updated to used in common file function actions.getMobileSession